MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj

```diff
@@ -262,4 +262,6 @@ This package provides you with the content pipeline for Windows, Mac and Linux w
     </Compile>
   </ItemGroup>
 
+  <Import Project="..\Stadia\Pipeline.targets" Condition="exists('..\Stadia\Pipeline.targets')"/>
```

Comment:
This seems like the right way to go for all future console support.  Lets us modify the project without it being visible to the public.